src/Imdb/Title.php

```diff
@@ -945,12 +945,17 @@ public function get_episode_details()
     public function plotoutline($fallback = false)
     {
         if ($this->main_plotoutline == "") {
-            $page = $this->getPage("Title");
-            if (preg_match('!class="summary_text">\s*(.*?)\s*</div>!ims', $page, $match)) {
-                $this->main_plotoutline = trim($match[1]);
-            } elseif ($fallback) {
-                $this->main_plotoutline = $this->storyline();
+            if (isset($this->jsonLD()->description)) {
+                $this->main_plotoutline = htmlspecialchars_decode($this->jsonLD()->description, ENT_QUOTES| ENT_HTML5);
+            } else {
+                $page = $this->getPage("Title");
```

Comment:
Is `$this->jsonLD()->description` not being there possible? If it's not this else block should be removed